sysl2/sysl/cmd_runner.go

```diff
@@ -11,6 +13,13 @@ import (
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
+const (
+	syslRootMarker          = ".sysl"
+	noRootMarkerWarning     = "%s is not defined but root flag is defined in %s"
+	rootMarkerExistsWarning = "%s found in %s but will use %s instead"
+	noRootWarning           = "root and %s are undefined, %s will be used instead"
```

Comment:
Move these down to where they are actually used, with them up here someone looking back at the warning code needs to scroll up and down to see what will be printed in what order. It also makes it more annoying to grep the source for the text (i.e you're running the program, see that text in the console and want to find where it comes from, its now a 2 step to get there).

There is also no value in making them const instead of just using them directly

Reply:
I'm using them in the test to check for logs. Should I copy the string to the test instead?

Reply:
im not a fan of testing for log messages TBH

Reply:
okay, removed log testings and constants, moved the warnings to where it is used